Templates/Tests/Expected/AutoGenerateProtocol.expected

```diff
@@ -0,0 +1,25 @@
+// Generated using Sourcery 0.10.1 — https://github.com/krzysztofzablocki/Sourcery
+// DO NOT EDIT
+
+
+//: Do not change this code as it is autogenerated every time you build.
+//: You can change the code in `../StencilTemplatesForSourcery/Application/AutoGenerateProtocol
+
+// MARK: - AutoGenerateProtocol
+//: From all Types implementing this protocol Sourcery adds:
+//: - public/internal variables // private variables are ignored
+//: - public/internal methods (skips initializers)
+//: - initializers marked with annotation // sourcery:includeInitInProtocol
+//: - of the above it does not add it if  // sourcery:skipProtocol
+//: ---
+
+// MARK: - Car AutoGenerated protocol
+protocol CarProtocol: AutoMockable {
```

Comment:
you should probably change either expected type or type in source. And you should actually test this template in `TemplatesTests.swift`